tests/integration/functional/cli/test_model.py

```diff
@@ -0,0 +1,326 @@
+    # Run the command
+    result = runner.invoke(list_registered_models, obj=concrete_model_registry)
+
+    # Check the result
+    assert result.exit_code == 0
+
+
+def test_update_model(
+    mocker,
+    concrete_registered_models,
+    concrete_model_registry,
+):
+    # Get the list command
+    runner = CliRunner()
+    update_registered_model = (
+        cli.commands["model-registry"].commands["models"].commands["update"]
+    )
+
+    # Patch the find_model_server method
+    mocker.patch.object(
+        ConcreteModelRegistry,
+        "update_model",
+        return_value=concrete_registered_models[0],
+    )
+    # Run the command
+    result = runner.invoke(
+        update_registered_model,
+        ["test_model", "--description", "new_test_description"],
+        obj=concrete_model_registry,
+    )
+
+    # Check the result
+    assert result.exit_code == 0
+
+
+def test_get_model(
+    mocker,
+    concrete_registered_models,
+    concrete_model_registry,
+):
+    # Get the list command
+    runner = CliRunner()
+    list_registered_model = (
+        cli.commands["model-registry"].commands["models"].commands["get"]
+    )
+
+    # Patch the find_model_server method
+    mocker.patch.object(
+        ConcreteModelRegistry,
+        "get_model",
+        return_value=concrete_registered_models[0],
+    )
+    # Run the command
+    result = runner.invoke(
+        list_registered_model, ["test_model"], obj=concrete_model_registry
+    )
+
+    # Check the result
+    assert result.exit_code == 0
+
+
+def test_list_model_versions(
+    mocker,
+    concrete_registered_model_version,
+    concrete_model_registry,
+):
+    # Get the list command
+    runner = CliRunner()
+    list_model_versions = (
+        cli.commands["model-registry"]
+        .commands["models"]
+        .commands["list-versions"]
+    )
+
+    # Patch the find_model_server method
+    mocker.patch.object(
+        ConcreteModelRegistry,
+        "list_model_versions",
+        return_value=concrete_registered_model_version,
+    )
+    # Run the command
+    result = runner.invoke(
+        list_model_versions, ["test_model"], obj=concrete_model_registry
+    )
+
+    # Check the result
+    assert result.exit_code == 0
+
+
+def test_get_model_version(
+    mocker,
+    concrete_registered_model_version,
+    concrete_model_registry,
+):
+    # Get the list command
+    runner = CliRunner()
+    get_model_version = (
+        cli.commands["model-registry"]
+        .commands["models"]
+        .commands["get-version"]
+    )
+
+    # Patch the find_model_server method
+    mocker.patch.object(
+        ConcreteModelRegistry,
+        "get_model_version",
+        return_value=concrete_registered_model_version[0],
+    )
+
+    # Run the command
+    result = runner.invoke(
+        get_model_version,
+        ["test_model", "--version", "test_version"],
+        obj=concrete_model_registry,
+    )
+    # Check the result
+    assert result.exit_code == 0
```

Comment:
same here. test that our model version is in `result.output`, and add another test to confirm the failure scenario (when no version exists).